libfreerdp/core/server.c

```diff
@@ -1693,9 +1697,9 @@ BOOL WINAPI FreeRDP_WTSVirtualChannelQuery(HANDLE hChannelHandle, WTS_VIRTUAL_CL
 						break;
 
 					default:
-						bval = FALSE;
-						status = FALSE;
-						break;
+						ppBuffer = NULL;
+						pBytesReturned = 0;
```

Comment:
Shouldn't this be `*ppBuffer = NULL;` and `*pBytesReturned = 0;`?

Reply:
oh good catch !